source/common/http/access_log/request_info_impl.h

```diff
@@ -46,6 +46,13 @@ struct RequestInfoImpl : public RequestInfo {
 
   void healthCheck(bool is_hc) override { hc_request_ = is_hc; }
 
+  void downStreamAddress(std::string address) override {downstream_address_ = address;}
+
+    /**
+     * Get the down stream address.
+     */
+   std::string downStreamAddress() const override {return downstream_address_;};
```

Comment:
const reference is because this is a getter and is not intended to change any member variable value on the class..